### PR TITLE
Link from crash message to the troubleshooting guide

### PR DESCRIPTION
**Pull request by bryk**
Re #1309

### CODE REVIEW

**Comment by bryk** [comment]
@rf232 @cheld @IanLewis Please take a look.

File: docs/user-guide/troubleshooting.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+<!--
+-----------------NOTICE------------------------
+This file is referenced in code as
+https://github.com/kubernetes/dashboard/blob/master/docs/user-guide/troubleshooting.md
+Do not move it without providing redirects.
```
Comment (IanLewis) [inline docs/user-guide/troubleshooting.md]: Are redirects possible on github?

Comment (rf232) [inline docs/user-guide/troubleshooting.md]: I think the project wiki is intended for these usecases, but not really sure how to set that up nicely

**Comment by IanLewis** [comment]
Thinking about it we could leave a file with a link to the new file if it's
ever moved. Not ideal but probably better than moving to the wiki.

**Comment by rf232** [comment]
Agreed, switching docu to wiki is not something we should do just for this usecase, and a non automatic HTTP 301 file sounds like a better solution :)

**Pull request merged**